eslint.config.js

```diff
@@ -14,20 +13,15 @@ export default [
                 ...globals.node,
                 ...globals.browser,
             },
+            parserOptions: {
```

Comment:
@simon-20 this is what was missing in order for it to be able to parse the Vue `import.meta.env` object which was initially erroring when I removed the typescript configs